src/js/components/MaskedInput/MaskedInput.js

```diff
@@ -113,7 +113,12 @@ const parseValue = (mask, value) => {
   return valueParts;
 };
 
-const defaultMask = [];
+const defaultMask = [
+  {
+    regexp: /[^]*/,
```

Comment:
I'm wondering if instead of setting default value we should enforce the prop to be a required field.